.gitignore

```diff
@@ -0,0 +1,2 @@
+# Ignore MD test file
+test.MD
```

Comment:
- [ ] Kindly add `node_modules/` to .gitignore file as you did with test.md file, to not upload all the packages to your repo.

Reply:
Ok done, thanks Kerolous.